_kbarticles/2022-12-08-common-kibana-searches-on-clouddotgov.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Common Kibana Searches on cloud.gov"
+date: December 8, 2022
+excerpt: This article documents a brief explaination of what Kibana does and common Kibana search queries on the cloud.gov platform.
+
+---
+
+## What does Kibana do?
+
+Application logs are visualized via Kibana, which has a [user guide](https://www.elastic.co/guide/en/kibana/current/index.html) that explains more about how to use it and customize your views.
+
+## What are some types of logs?
+
+Cloud Foundry assigns a type to each log message depending on its origin. Application logs are assigned the APP log type.  HTTP requests being routed to an app will produce the RTR log type. The various types of logs are listed in the documentation [here](https://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html#format).
```

Comment:
```suggestion
Cloud Foundry assigns a type to each log message depending on its origin. Application logs are assigned the APP log type.  HTTP requests being routed to an app will produce the RTR log type. The various types of logs are listed in the documentation [here](https://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html#format).

Cloudfoundry logs are often translated into field names in Elasticsearch/Kibana using the log type as a prefix. For example, APP logs in Kibana include fields like `app.name` for the application name.
```

I think it's helpful to give some idea of how CloudFoundry log types are translated to searchable fields in Kibana